reducer/src/test/java/com/graphicsfuzz/reducer/reductionopportunities/RemoveRedundantUniformMetadataReductionOpportunitiesTest.java

```diff
@@ -82,14 +74,12 @@ public void testDoNotRemoveUsed() throws Exception {
     assertEquals(1, shaderJob.getPipelineInfo().getNumUniforms());
 
     // There should be exactly one opportunity to remove a piece of unused pipeline state.
```

Comment:
There was a typo in this comment; it should read "There should be no opportunities to ..." - please fix in this PR.

Reply:
Fixed